### PR TITLE
Removing the :reader feature toggle

### PR DESCRIPTION
**Pull request by mdbenjam**
Connects: #3230 

We have turned off whitelisting in prod, so we can remove the code to check for the feature toggle.

**Test Plan**
- [ ] Make sure we can still access Reader with or without the feature toggle.

### CODE REVIEW

File: app/controllers/reader/application_controller.rb [hunk 1]
```diff
@@ -1,22 +1,10 @@
 class Reader::ApplicationController < ApplicationController
-  before_action :verify_access, :verify_reader_feature_enabled, :react_routed, :check_reader_out_of_service
-
-  def logo_name
```
Comment (mdbenjam) [inline app/controllers/reader/application_controller.rb]: These are no longer needed after moving the header rendering from Rails -> React. They should've been taken out before, but now it's complaining about test coverage, so they have to come out.

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -171,14 +171,6 @@ def page_title(title)
   end
   helper_method :page_title
 
-  def verify_feature_enabled(feature)
```
Comment (mdbenjam) [inline app/controllers/application_controller.rb]: This is no longer being called.

**Comment by nickheiner-usds** [comment]
> My favorite kind of PR! (-48 lines, 1 addition)

:smile:

File: db/seeds.rb [hunk 3]
```diff
@@ -109,11 +109,6 @@ def clean_db
     DatabaseCleaner.clean_with(:truncation)
   end
 
-   def set_up_feature_toggles
```
Comment (joofsh) [inline db/seeds.rb]: @mdbenjam this broke running `rake db:seed`

screen shot 2017-09-11 at 5 57 50 pm

Can you re-add the empty method, or delete its reference?

Comment (mdbenjam) [reply db/seeds.rb]: gah yeah give me a minute

Comment (sunil-sadasivan) [reply db/seeds.rb]: Should have tested the db seeder when reviewing!  Apologies 🙈

**Pull request merged**